idris-warnings-tree.el

```diff
@@ -38,18 +38,19 @@
 (defvar idris-tree-printer 'idris-tree-default-printer)
 
 (defun idris-compiler-notes-list-show (notes)
```

Comment:
Considering renaming this function as it handles full life cycle of the notes not just showing them.